### DISCUSSION TITLE
Where can I remove WebRTC from source?

### DISCUSSION THREAD

**Discussion by owly111** [cefsharp/CefSharp#3523] [Q&A]
Hello, I'm trying to disable WebRTC as it leaks my real ip address. I tried every configuration for settings initialization but with no success. I found this thread: https://magpcss.org/ceforum/viewtopic.php?f=14&t=15194 where they say:
`The only reliable way to completely disable WebRTC is do private CEF build with GN_DEFINE [c]enable_webrtc=false[/c].
`

I am searching through the source code with no success. Thank you in advance

**Comment by amaitland** [comment]
Try ask on https://magpcss.org/ceforum/index.php

From memory there are a number of similar discussions. Questions related to building `CEF` should be asked on ceforum.

**Reply by owly111** [reply]
Thank you
